doc/sphinx/source/community/diagnostic.rst

```diff
@@ -45,15 +45,25 @@ Always make sure your code is or can be released under a license that is compati
 
 If you have existing code in a supported scripting language, you have two options for re-using it. If it is fairly
 mature and a large amount of code, the preferred way is to package and publish it on the
-official package repository for that language and add it as a dependency of esmvaltool.
+official package repository for that language and add it as a dependency of ESMValTool.
 If it is just a few simple scripts or packaging is not possible (i.e. for NCL) you can simply copy
-and paste the source code into the esmvaltool/diag_scripts directory.
+and paste the source code into the ``esmvaltool/diag_scripts`` directory.
 
 If you have existing code in a compiled language like
 C, C++, or Fortran that you want to re-use, the recommended way to proceed is to add Python bindings and publish
 the package on PyPI so it can be installed as a Python dependency. You can then call the functions it provides
 using a Python diagnostic.
 
+Additional dependencies
```

Comment:
I think this section is not specific to diagnostic scripts, so it would fit better introduction.rst

Reply:
It sort of is specific to diagnostic scripts, because that will be the bulk of code that will be contributed and it is listed in that part of the checklist. I added it below the section where it talks about re-using code, and additional dependencies logically belongs there. There is no such a section in the `introduction.rst`, so I prefer to keep it here.

Reply:
The CMORizer scripts also have their own dependencies

Reply:
Then the section about re-using code could also move elsewhere.

Reply:
I guess so, though I do think that's a much more likely scenario for diagnostics than it is for CMORizers, because the latter are typically quite small scripts

Reply:
Let's tackle it then in a seperate PR where we look more critically at the structure of the contributing guidelines.

Reply:
I think I would leave the dependencies here for now. The whole CMORization is kind of changing right now, I think, with the pull request #1657.